src/applications/appeals/995/tests/validations/validations.unit.spec.js

```diff
@@ -87,6 +87,14 @@ describe('contactInfoValidation', () => {
     expect(addError.secondCall.args[0]).to.eq(errorMessages.missingPhone);
     expect(addError.lastCall.args[0]).to.eq(errorMessages.missingAddress);
   });
+  it('should not throw an error when addError function is missing', () => {
+    try {
+      contactInfoValidation();
+      expect(true).to.be.true;
+    } catch (error) {
+      expect(error).to.be.null;
```

Comment:
On the catch part, how do we expect `error` to be `null` if we expect this to not throw an error when the `addError` function is missing?

Reply:
It's like you said in the next comment, think of the `try... catch` as an `if... else` - the code within the `try` will execute completely unless there is an error thrown, then the code within the `catch` will execute. Within the `catch`, the `error` will be defined, so the `expect(error).to.be.null` will fail and cause the unit test to fail - which shouldn't happen since we added optional chaining to prevent an undefined function from being executed.